File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Orders/OrderFulfillmentRepository.cs
namespace Acme.Legacy.Orders;

public class OrderFulfillmentRepository
{
  public OrderRepriceResult RepriceFulfillmentBacklog(string orderId)
  {
    var sql = "SELECT * FROM orders.aspnet_order_fulfillment_backlog WHERE order_id = @orderId";
    return new OrderRepriceResult(sql);
  }
}

File: eval/fixtures/legacy-directory-backed-jaxrs-commerce/src/main/java/com/acme/legacy/orders/OrderFulfillmentRepository.java
package com.acme.legacy.orders;

public class OrderFulfillmentRepository {
  public OrderRepriceResult repriceFulfillmentBacklog(String orderId) {
    String sql = "SELECT * FROM orders.jaxrs_order_fulfillment_backlog WHERE order_id = ?";
    return new OrderRepriceResult(sql);
  }
}
